baremetal/metal3data_manager_test.go

```diff
@@ -91,7 +99,7 @@ var _ = Describe("Metal3Data manager", func() {
 		expectedErrorSet bool
 	}
 
-	DescribeTable("Test CreateSecret",
+	DescribeTable("Test Reconcile",
```

Comment:
Test is for the `Reconcile` function and is using `testCaseReconcile` type. The name was just wrong